administration/src/bp-modules/home/HomeController.tsx

```diff
@@ -53,6 +53,13 @@ const HomeController = () => {
           <StyledButton icon='path-search' text='Region verwalten' />
         </NavLink>
       ) : null}
+      {role === Role.ProjectAdmin || (role === Role.RegionAdmin && cardStatistics.enabled) ? (
+        <>
+          <NavLink to={'/statistics'}>
```

Comment:
The "statistics" item is duplicated (apparently a merge issue?)

Reply:
probably thats the issue. i removed that